apps/prairielearn/src/lib/question-render.ts

```diff
@@ -627,7 +691,10 @@ export async function renderPanelsForSubmission({
       panels.answerPanel = locals.showTrueAnswer ? htmls.answerHtml : null;
       panels.extraHeadersHtml = htmls.extraHeadersHtml;
 
-      await manualGrading.populateRubricData(locals);
+      const rubric_data = await manualGrading.buildRubricData({
+        assessment_question,
+        submission,
+      });
```

Comment:
There was an actual bug here: `populateRubricData` relied on `locals.assessment_question` being present, but it was never assigned above this. It's easy to reproduce the problem:

- In student mode onn a manually-graded question, make a submission.
- In instructor mode, manually grade that submission with a rubric.
- Back in student mode, make at least three more submissions to that question.
- Click "show/hide older submissions".
- Expand the teal feedback panel; observe that it shows the rubric grading.
- Click to expand the submission itself; observe that the teal feedback panel no longer has any content.